synapse/lib/stormtypes.py

```diff
@@ -7919,6 +7927,28 @@ async def _methViewFork(self, name=None):
 
         return View(self.runt, newv, path=self.path)
 
+    async def _methViewInsertParentFork(self, name=None):
+        useriden = self.runt.user.iden
+        viewiden = self.valu.get('iden')
+
+        if name is not None:
+            name = await tostr(name)
```

Comment:
small nit, but couldn't you just do tostr(name, noneok=True)?

Reply:
good point, yeah